variables/all.yaml

```diff
@@ -0,0 +1,8 @@
+manifest_version: "release"
```

Comment:
Where do you use this file?

Reply:
https://github.com/kubevirt/kubevirt-ansible/pull/67/files#diff-d32cf98e95fc322fdf933a2d2b6724a6